helm-install/install-helm.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Just to clarify - how is this file supposed to be used? Manual copy-paste from our repo for installation or this will automatically be picked up by Artifactory?

Reply:
No, this is for locally spinning up HELM for testing, and also can serve as an example on how to do it.

Reply:
It's better to explain it in a comment as it won't be entirely clear to others in future.